charts/consul/test/terraform/aks/main.tf

```diff
@@ -48,7 +55,7 @@ resource "azurerm_kubernetes_cluster" "default" {
   location                          = azurerm_resource_group.default[count.index].location
   resource_group_name               = azurerm_resource_group.default[count.index].name
   dns_prefix                        = "consul-k8s-${random_id.suffix[count.index].dec}"
-  kubernetes_version                = "1.24.6"
+  kubernetes_version                = "1.26"
```

Comment:
Should this be set to a minor version? Or are you thinking that if we keep at 1.26 we will be ok until the next upgrade?